muxink/Cargo.toml

```diff
@@ -12,9 +12,11 @@ serde = { version = "1.0.138", optional = true }
 thiserror = "1.0.31"
 tokio = { version = "1.18.1", features = [ "full" ] }
 tokio-util = "0.7.2"
+casper-types = { path = "../types", optional = true }
 
 [dev-dependencies]
 tokio-stream = "0.1.8"
 
 [features]
-bincode = [ "dep:serde", "dep:bincode" ]
+muxink_bincode_codec = [ "serde", "bincode" ]
+muxink_bytesrepr_codec = [ "casper-types" ] 
```

Comment:
These two features should be excluding one another, right? I.e. it doesn't make sense to have both bincode and bytesrepr at the same time?

Reply:
It's conceivable to actually use both, they're just encodings. We may even start with `bincode` only, then transfer ACK streams to `ToBytes` first, keeping others using `bincode` and finally do the switch.